thirdparty/cmdconfig/commands/cmdsink/cmdsink_test.go

```diff
@@ -5,21 +5,25 @@ package cmdsink
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/GoogleContainerTools/kpt/internal/printer"
 	"github.com/GoogleContainerTools/kpt/internal/printer/fake"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestSinkCommand(t *testing.T) {
```

Comment:
These tests are closer to `e2e` tests. It will be better if we can just replace them with `e2e` tests.

Reply:
These are standalone tests for sink command. It is good to have them to understand sink behavior in isolated fashion. Doesn't harm having them.